src/script/descriptor.cpp

```diff
@@ -262,6 +262,16 @@ class BIP32PubkeyProvider final : public PubkeyProvider
         return true;
     }
 
+    // Derives the last xpub
```

Comment:
`xpub or xpriv`, unless you neuter it at the end and have it return a `CExtPubKey&`

Reply:
Changed to xprv